src/Locales/en-gb/Pode.psd1

```diff
@@ -0,0 +1,285 @@
+    cannotCreatePropertyWithoutTypeExceptionMessage                       = 'Cannot create the property because no type is defined.'
+    authMethodNotExistForMergingExceptionMessage                          = 'Authentication method does not exist for merging: {0}'
+    maxValueInvalidExceptionMessage                                       = "Max value '{0}' for {1} is invalid, should be less than/equal to {2}"
+    endpointAlreadyDefinedExceptionMessage                                = "An endpoint named '{0}' has already been defined."
+    eventAlreadyRegisteredExceptionMessage                                = '{0} event already registered: {1}'
+    parameterNotSuppliedInRequestExceptionMessage                         = "A parameter called '{0}' was not supplied in the request or has no data available."
+    cacheStorageNotFoundForSetExceptionMessage                            = "Cache storage with name '{0}' not found when attempting to set cached item '{1}'"
+    methodPathAlreadyDefinedExceptionMessage                              = '[{0}] {1}: Already defined.'
+    errorLoggingAlreadyEnabledExceptionMessage                            = 'Error Logging has already been enabled.'
+    valueForUsingVariableNotFoundExceptionMessage                         = "Value for '`$using:{0}' could not be found."
+    rapidPdfDoesNotSupportOpenApi31ExceptionMessage                       = "The Document tool RapidPdf doesn't support OpenAPI 3.1"
+    oauth2ClientSecretRequiredExceptionMessage                            = 'OAuth2 requires a Client Secret when not using PKCE.'
+    invalidBase64JwtExceptionMessage                                      = 'Invalid Base64 encoded value found in JWT'
+    noSessionToCalculateDataHashExceptionMessage                          = 'No session available to calculate data hash.'
+    cacheStorageNotFoundForRemoveExceptionMessage                         = "Cache storage with name '{0}' not found when attempting to remove cached item '{1}'"
+    csrfMiddlewareNotInitializedExceptionMessage                          = 'CSRF Middleware has not been initialised.'
+    infoTitleMandatoryMessage                                             = 'info.title is mandatory.'
+    typeCanOnlyBeAssociatedWithObjectExceptionMessage                     = 'Type {0} can only be associated with an Object.'
+    userFileDoesNotExistExceptionMessage                                  = 'The user file does not exist: {0}'
+    routeParameterNeedsValidScriptblockExceptionMessage                   = 'The Route parameter needs a valid, not empty, scriptblock.'
+    nextTriggerCalculationErrorExceptionMessage                           = 'Looks like something went wrong trying to calculate the next trigger datetime: {0}'
+    cannotLockValueTypeExceptionMessage                                   = 'Cannot lock a [ValueTypes]'
```

Comment:
This can just be `[ValueType]`, to avoid confusion with `[ValueTypes]` which doesn't exist :) unsure if this is the same in other translations.